app.py

```
@@ -1,36 +1,88 @@
 import streamlit as st
 from pathlib import Path
 import json
+import os
+import shutil
+
 # For some reason the windows version only works if this is imported here
 import pyopenms
```

Ruff (0.8.2)

`pyopenms` imported but unused

Remove unused import: `pyopenms`

(F401)